Assets/Levels/FirstPersonMovementTest/FirstPersonMovementTest/NavMesh.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 5d9ac3d3160b05f498a41a75d278dc87
```

Comment:
added for enemy to interact with the ground beneath and for path finding (meta file)